### DISCUSSION TITLE
Can I install TTS on a Raspberry Pi 4?

### DISCUSSION THREAD

**Discussion by 621625** [coqui-ai/TTS#1569] [General Q&A]
Hello,

I'm BM. A very nice and polite guy.

Is it possible to install TTS on a Raspberry Pi 4? Most of the dependencies install well but at the end of the installation I get this message about mecab-python3 and also an error message about llvmlite. Any workaround?

Building wheels for collected packages: mecab-python3, llvmlite
  Building wheel for mecab-python3 (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-8_jufamh/mecab-python3_69eb6ee330714cdbb17c4fdde7259ec7/setup.py'"'"'; __file__='"'"'/tmp/pip-install-8_jufamh/mecab-python3_69eb6ee330714cdbb17c4fdde7259ec7/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-8t2_jjyn
       cwd: /tmp/pip-install-8_jufamh/mecab-python3_69eb6ee330714cdbb17c4fdde7259ec7/
  Complete output (10 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-aarch64-3.9
  creating build/lib.linux-aarch64-3.9/MeCab
  copying src/MeCab/__init__.py -> build/lib.linux-aarch64-3.9/MeCab
  copying src/MeCab/cli.py -> build/lib.linux-aarch64-3.9/MeCab
  running build_ext
  error: [Errno 2] No such file or directory: 'mecab-config'
  ----------------------------------------
  ERROR: Failed building wheel for mecab-python3
  Running setup.py clean for mecab-python3
  Building wheel for llvmlite (setup.py) ... error

**Comment by erogol** [comment]
You might need to strip some of the requirements that are not Raspi capable.

**Comment by ludwikbukowski** [comment]
I managed to install on x64

**Reply by 621625** [reply]
That's not the same as ARM64, right? Or is it? If so, please provide instructions if possible.

**Reply by fquirin** [reply]
I've made a Docker container for aarch64/arm64 with Python 3.10, you can check out the build script if you like: https://hub.docker.com/r/sepia/coqui-tts
What I noticed is that there seems to be a problem with Python 3.9 on Arm64: https://github.com/coqui-ai/TTS/issues/2351

**Comment by 621625** [comment]
Error message:

sudo docker run --rm --name=coqui_tts -p 59125:80 -it sepia/coqui-tts:aarch64 
docker: invalid reference format.
See 'docker run --help'.


------- Original Message -------

**Reply by fq-selbach** [reply]
The exact same command is working for me 🤔 , maybe a copy-paste error?

'[docker: invalid reference format.](https://techoverflow.net/2019/10/14/how-to-fix-docker-invalid-reference-format/)' usually means there is a formatting error in your command, maybe an unintentional line-break?
